django/contrib/admin/static/admin/css/responsive.css

```diff
@@ -163,10 +163,6 @@ input[type="submit"], button {
         margin-right: 230px;
     }
 
-    #changelist .paginator {
-        border-top-color: #eee;
-    }
```

Comment:
Thanks! I'm not sure whether the removal was intentional or not. I added it back with a comment so that I can check out later whether the rule has any effect or not.

It's sometimes hard to see what's even used, not only but also because the CSS contains admindocs rules too (e.g. `blockquote` in `base.css`)